gulp-tasks/jekyll.js

```diff
@@ -1,11 +1,28 @@
 'use strict';
 
+const fse = require('fs-extra');
 const gulp = require('gulp');
 const path = require('path');
 const spawn = require('child_process').spawn;
 
+
+const addNpmDependencies = () => {
```

Comment:
I'm assuming the hardcoded `/` path separators are going to cause issues on Windows. Can you switch to `path.join()`?